reana_workflow_controller/config.py

```diff
@@ -48,6 +48,12 @@
 }
 """Mapping from the shared file system backend to the job file system."""
 
+REANA_JOB_STATUS_CONSUMER_PREFETCH_COUNT = int(
+    os.getenv("REANA_JOB_STATUS_CONSUMER_PREFETCH_COUNT", 200)
+)
+"""Prefetch count value defines max number of messages that will be delivered to
+the ``jobs-status`` queue consumer."""
```

Comment:
*nitpick*: will be good to make it clear that messages that are prefetched are unacknowledged 

```suggestion
"""The value defines the max number of unacknowledged deliveries that are permitted on a ``jobs-status`` consumer."""
```

Taken from [RabbitMQ docs](https://www.rabbitmq.com/confirms.html#channel-qos-prefetch)